pallets/pool-system/src/pool_types.rs

```diff
@@ -332,3 +332,10 @@ impl<
 		})
 	}
 }
+
+#[derive(Encode, Decode, Clone, Eq, PartialEq, RuntimeDebug, Default, TypeInfo, MaxEncodedLen)]
+pub struct ExtPoolChange {
+	submitted_time: Moment,
+	delay_time_required: Moment,
+	block_by_locked_redemptions: bool,
+}
```

Comment:
Maybe would be more extensible if this was
```suggestion

pub enum ExtPoolChangeRequirements {
  DelayTime(Moment),
  BlockedByLockedRedemptions,
}

#[derive(Encode, Decode, Clone, Eq, PartialEq, RuntimeDebug, Default, TypeInfo, MaxEncodedLen)]
pub struct ExtPoolChange {
	submitted_time: Moment,
	requirements: BoundedBTreeSet<ExtPoolChangeRequirements, RequirementsSize>,
}
```

Reply:
I still do not know well what the requirements will be, but I like this very much!

Do you know another future requirement as an example?